### DISCUSSION TITLE
Only enable debugger and turn off everything else

### DISCUSSION THREAD

**Discussion by wshv** [microsoft/vscode-cpptools#11198] [Q&A]
Is there a way to turn off every feature shipped with this extension (e.g. language server, formatted, linter) except for the debugger?

This is the current setting I’m using and I’m not sure if it is enough or not
```
    "C_Cpp.codeAnalysis.runAutomatically": false,
    "C_Cpp.autocomplete": "disabled",
    "C_Cpp.suggestSnippets": false,
    "C_Cpp.formatting": "disabled",
    "C_Cpp.errorSquiggles": "disabled",
```

Thanks in advance

**Comment by sean-mcmanus** [comment]
@wshv Set C_Cpp.intelliSenseEngine to "disabled". I don't think you need to change the other settings.
